Homework3/StackCalculatorTests/StackCalculator.Tests/StackCalculatorTests.cs

```diff
@@ -0,0 +1,73 @@
+namespace StackCalculatorTask.Tests;
+
+public class StackCalculatorTests
+{
+    StackCalculator calculator = new StackCalculator();
+    
+    private static IEnumerable<IStack> stacks
```

Comment:
```suggestion
    private static IEnumerable<IStack> Stacks
```
Свойства именуются с заглавной, даже если они private